app/models/alert.rb

```diff
@@ -30,6 +31,7 @@ class Alert < ApplicationRecord
 
   scope :violations, -> { where.has { level >=  VIOLATION_LEVEL }  }
   scope :alerts, -> { where.has { level < VIOLATION_LEVEL } }
+  scope :stale, -> { where.has { created_at <= TTL.ago } }
```

Comment:
I'm getting 
```
Error performing PurgeStaleObjectsWorker (Job ID: 1b9dd182-59f0-4021-ad04-b3df2892922c) from Sidekiq(default) in 49.93ms: BabySqueel::NotFoundError (There is no function or column or association named 'created_at' for Alert.):
```

And indeed, I can't see `created_at` or `updated_at` timestamp fot ehr `alerts` table :thinking:

Reply:
What we do have is `timestamp` which is, I guess, the moment when the event happened on the backend.

Reply:
I hate synthetic unit tests tests :( I'll check it out once we have an approval for implementing this. No need before we decide what to do exactly.